omod-common/pom.xml

```diff
@@ -31,9 +31,9 @@
 		</dependency>
 
 		<dependency>
-			<groupId>io.swagger</groupId>
+			<groupId>io.swagger.core.v3</groupId>
```

Comment:
Quick thing: since the versions are already defined in the root POM (in dependency management), they don't need to be specified here. That way we only need to update versions in a single place (dependency management doesn't make something a dependency, but it's configuration applies to a dependency with the same coordinates.

For the same reasons, we shouldn't need to replicate the exclusions here.

Reply:
fixed